tasks/http.md

```diff
@@ -73,6 +73,8 @@ Add answers to the following questions to your repository:
 
 Choose http verbs for every piece of functionality. If applicable, describe also the way you would provide data to the request (vie query params, in request body etc.). If applicable, describe the response of the server. Explain your choice.
 
+You did lot already! If you honestly finished all the previous steps then go ahead and share it with others – post a message in course channel: Http & Https — #done (or TCP. UDP. Network — #done if you are p2p course student) and add the link to your repo. This step is important, as it helps mentors to track your progress! When complete do the following:
```

Comment:
do we need "When complete do the following:" at the end?
or I miss something?

Reply:
Next is a section with Extra Materials like in previous chapter
https://github.com/kottans/backend/blob/master/tasks/network.md
maybe its a mistake in the previous chapter, don't know

Reply:
Maybe it can be changed to "When you finish this task you can proceed to the next one." as in https://github.com/kottans/backend/blob/master/tasks/memory-management.md ?
What do you think?

Reply:
sounds good.
changed